lib/assets/javascripts/_modules/search.js

```diff
@@ -65,7 +67,7 @@
       // We need JavaScript to do search, so if JS is not available the search
       // input sends the query string to Google. This JS function changes the
       // input to instead send it to the search page.
-      $searchForm.prop('action', '/search')
+      $searchForm.prop('action', pathToSiteRoot + 'search/index.html')
```

Comment:
I think it's best if this doesn't have the `index.html` file name, to be consistent with other URLs. Is there a reason that you changed it?

```suggestion
      $searchForm.prop('action', pathToSiteRoot + 'search/')
```

Reply:
If we don't include the `index.html` then I _think_ we become reliant on the web server that the site is deployed behind being configured to have a default page value of `index.html`. Having said that I think there are other areas where we generate links which rely on the default page behaviour, so you could argue this is incongruous with that approach. Happy to try taking it out if you'd prefer consistency here.